gam_app/static/js/document_page.js

```diff
@@ -92,3 +84,31 @@ function isPointInsideRect(point, rect) {
     return point.x > rect.x && point.x < rect.x + rect.width && point.y > rect.y &&
            point.y < rect.y + rect.height;
 }
+
+function areAllFullyLoaded() {
+  var tiledImage;
+  var count = viewer.world.getItemCount();
+  for (var i = 0; i < count; i++) {
+    tiledImage = viewer.world.getItemAt(i);
+    if (!tiledImage.getFullyLoaded()) {
+      return false;
+    }
+  }
+  return true;
+}
+
+var isFullyLoaded = false;
+
+viewer.world.addHandler('add-item', function(event) {
+  var tiledImage = event.item;
+  tiledImage.addHandler('fully-loaded-change', function() {
+    var newFullyLoaded = areAllFullyLoaded();
+    if (newFullyLoaded !== isFullyLoaded) {
```

Comment:
I think this whole thing could just be simplified to `if (areAllFullyLoaded()) {`. It's not like we ever use the `isFullyLoaded` variable anywhere else. Try that and see if it works.